### PR TITLE
German: Suggested corrections for ability.txt

### PR DESCRIPTION
**Pull request by Talandazar**

### CODE REVIEW

File: crawl-ref/source/dat/descript/de/ability.txt [hunk 1]
```diff
@@ -822,54 +819,53 @@ nicht rückgängig gemacht werden.
...
 Heavenly Storm ability
 
-Ruft einen Sturm aus himmlischen Wolken an. Während der Sturm wütet, werden
-Eure Ausweich- und Kampffähigkeit sowie Euer Schaden erheblich erhöht, und
-das Treffen von Feinden mit martialischen Angriffen erhöht diesen Bonus. Der
-Sturm wird mit der Zeit schwächer und verschwindet, sobald der Bonus Null
-erreicht. Die himmlischen Wolken verhindern Unsichtbarkeit und verschleiern die
+Ruft einen Sturm aus himmlischen Wolken hervor. Während der Sturm wütet, werden
+Euer Ausweichen, Eure Treffsicherheit sowie Euer Schaden erheblich erhöht und
+wenn ihr Feinde mit martialischen Angriffen trefft, erhöht sich der Bonus. Der
```
Comment (powerbf) [inline crawl-ref/source/dat/descript/de/ability.txt:852]: I think this should be Ihr

File: crawl-ref/source/dat/descript/de/ability.txt [hunk 2]
```diff
@@ -682,110 +681,108 @@ Ihr Ränge in der Fertigkeit Evozieren habt.
...
 Monster versetzen Euch in Schrecken, was die Macht Eurer Zauber und Eure
-Fähigkeit, Monster im Kampf zu treffen und ihnen Schaden zuzufügen,
-verringert.
+Fähigkeit verringert, Monster im Kampf zu treffen und ihnen Schaden zuzufügen.
 %%%%
 Sacrifice Arcana ability
 
 Opfert Arkana, wodurch Ihr die Fähigkeit verliert, bestimmte Magieschulen zu
-benutzen und keine Ränge in diesen Fertigkeiten behalten könnt.
+benutzen und alle Ränge in diesen Fertigkeiten verliert.
```
Comment (powerbf) [inline crawl-ref/source/dat/descript/de/ability.txt:697]: You don't just lose current ranks. You also can't train the skill anymore. Same goes for the next two sacrifices.

File: crawl-ref/source/dat/descript/de/ability.txt [hunk 3]
```diff
@@ -145,66 +144,66 @@ Lässt Euch in Eure normale Form zurückkehren.
...
 zentriert ist und Untoten und Dämonen schweren Schaden zufügt sowie allen
 anderen feindlichen Kreaturen Schaden zufügt. Die Kraft der Explosion wird
 durch die Fertigkeit Anrufungen erhöht.
 %%%%
 Summon Divine Warrior ability
 
 Ruft einen mächtigen Krieger gegen das Böse herbei, entweder einen Engel oder
-einen Daeva. Die Dauer ihres Beistands wird durch die Fertigkeit Anrufungen
-erhöht, und Ruhm im Kampf kann dazu führen, dass der Glänzende sie mit einem
+eine Daeva. Die Dauer ihres Beistands wird durch die Fertigkeit Anrufungen
```
Comment (powerbf) [inline crawl-ref/source/dat/descript/de/ability.txt:198]: Ah, Daeva is feminine. Thanks. I will correct this in the other text files.

File: crawl-ref/source/dat/descript/de/ability.txt [hunk 4]
```diff
@@ -145,66 +144,66 @@ Lässt Euch in Eure normale Form zurückkehren.
...
+erhöht und Ruhm im Kampf kann dazu führen, dass der Strahlende sie mit einem
 noch längeren Aufenthalt segnet.
 %%%%
 Brand Weapon With Holy Wrath ability
 
-Segnet eine Waffe Eurer Wahl dauerhaft mit dem heiligen Zorn des Glänzenden.
+Segnet eine Waffe Eurer Wahl dauerhaft mit dem heiligen Zorn des Strahlenden.
```
Comment (powerbf) [inline crawl-ref/source/dat/descript/de/ability.txt:204]: I will also replicate this change in other text files.

File: crawl-ref/source/dat/descript/de/ability.txt [hunk 5]
```diff
@@ -216,100 +215,101 @@ Anzahl und Frische der Leichen hängt von der Nekromantie-Fertigkeit ab.
...
+zuzufügen, die jenes Wesen in seinem Leben erlitten hat. Dieser Schaden ist
+niemals direkt tödlich. Kikubaaqudgha kann Euch vor diesem Effekt schützen,
+je nach Eurer Frömmigkeit.
+.
 %%%%
 Brand Weapon With Pain ability
 
-Lässt eine Waffe Eurer Wahl dauerhaft mit der Essenz des Schmerzes selbst
-bluten. Dieser Segen kann nicht auf Artefaktwaffen angewendet werden. Ihr
+Besudelt eine Waffe Eurer Wahl dauerhaft mit der Essenz des Schmerzes.
```
Comment (powerbf) [inline crawl-ref/source/dat/descript/de/ability.txt:225]: Nice.

File: crawl-ref/source/dat/descript/de/ability.txt [hunk 6]
```diff
@@ -569,55 +569,55 @@ Träger von dem Fluch.
 Shadow Step ability
 
 Erlaubt es Euch, in den Schatten einer beliebigen feindlichen Kreatur innerhalb
-Eures Umbras zu treten und an einem zufälligen Ort in der Nähe des Ziels zu
+Eurer Umbra zu treten und an einem zufälligen Ort in der Nähe des Ziels zu
```
Comment (powerbf) [inline crawl-ref/source/dat/descript/de/ability.txt:572]: Thanks. I will correct the gender of Umbra in the other text files.

File: crawl-ref/source/dat/descript/de/ability.txt (hunk 1 above)
Comment (Talandazar) [reply crawl-ref/source/dat/descript/de/ability.txt:852]: Yes, just a typo.

File: crawl-ref/source/dat/descript/de/ability.txt (hunk 2 above)
Comment (Talandazar) [reply crawl-ref/source/dat/descript/de/ability.txt:697]: Looked at the English version again. The most literal translation would be:
+ benutzen und keine Ränge in diesen Fertigkeiten haben könnt.

That covers both aspects.

File: crawl-ref/source/dat/descript/de/ability.txt [hunk 7]
```diff
@@ -653,26 +653,25 @@ Opfert die Reinheit von Körper oder Geist und verursacht eine einzige Mutation.
...
 Sacrifice Essence ability
 
-Opfert die magische Essenz und verursacht eine einzige Mutation.
+Opfert magische Essenz und verursacht eine einzige Mutation.
 %%%%
 Sacrifice Health ability
 
-Opfert die körperlichen Gesundheit, wodurch eine einzige Mutation verursacht
-wird.
+Opfert körperlichen Gesundheit und verursacht eine einzige Mutation.
```
Comment (Talandazar) [inline crawl-ref/source/dat/descript/de/ability.txt:669]: That's an oversight. With or without 'die', it should be 'körperliche'. The n is wrong in both cases. The 'die' isn't exactly wrong, it just sounds better without it.

**Review by powerbf — CHANGES_REQUESTED** [review]
We should update the skill sacrifice thing.

**Review by powerbf — APPROVED** [review]

**Pull request merged**